### PR TITLE
Exposing internal context

### PR DESCRIPTION
**Pull request by StLyn4**
**What is the purpose of this pull request? (put an "X" next to item)**

[  ] Documentation update
[  ] Bug fix
[X] New feature
[  ] Other, please explain:

**What changes did you make? (Give an overview)**
Allows users to access context with translations. What for? The investigation showed that during page transition animation, the context is updated too early. This problem appears to be due to the nature of libraries such as `framer-motion`. Context management is located above animation management. From possible workarounds, this one has shown itself to be the most stable so far (of course, it cannot be 100% guaranteed that it will work without bugs. But at least I have not observed them yet): https://github.com/aralroca/next-translate/issues/513#issuecomment-1272525899
But it needs access to the context. Actually, for what this PR was created. The fork has been in production for a few months and hasn't shown any problems so far.

Basically, this update just gives access to one internal variable. No more no less.

**Which issue (if any) does this pull request address?**
#513

### CODE REVIEW

**Review by aralroca — APPROVED** [review]
Looks good to me! Thank you!
